02-04-2015/coins/coins_subhash.txt

```diff
@@ -0,0 +1,11 @@
+class Coins
+
+  COIN_VALUES = [10, 5, 2, 1, 0.5, 0.25]
+
+  def self.count(num)
+    return 0 if num == 0
+    min_count = 0
+    current_coin = COIN_VALUES.select{|c| c if c <= num}.first
```

Comment:
As per doc
> `Enumerable#find` Passes each entry in enum to block. Returns the first for which block is not false.

So I thought it would be efficient that it doesn't traverse entire array.